ch01python/016functions.ipynb.py

```diff
@@ -28,100 +28,86 @@
 def double(x):
     return x * 2
 
-print(double(5), double([5]), double('five'))
+
+# %%
+double(5)
+
+# %%
+double([5])
+
+# %%
+double("five")
 
 
 # %% [markdown]
-# ### Default Parameters
+# ## Default Parameters
 
 # %% [markdown]
 # We can specify default values for parameters:
 
 # %%
-def jeeves(name = "Sir"):
-    return f"Very good, {name}"
+def jeeves(name="Sir"):
+    return "Very good, " + name
```

Comment:
f-strings are more efficient
```suggestion
def jeeves(name="Sir"):
    return f"Very good, {name}"
```